torchtext/experimental/datasets/raw/question_answer.py

```diff
@@ -28,32 +29,56 @@ def _create_data_from_json(data_path):
                     yield (_context, _question, _answers, _answer_start)
 
 
-def _setup_datasets(dataset_name, root='.data'):
+def _setup_datasets(dataset_name, root='.data', data_select=('train', 'dev')):
+    data_select = check_default_set(data_select, ('train', 'dev'))
     extracted_files = []
     select_to_index = {'train': 0, 'dev': 1}
     extracted_files = [download_from_url(URLS[dataset_name][select_to_index[key]],
```

Comment:
You could edit the URLs to actually be a dictionary instead of having this map.

Reply:
Fixed.